ArcadiaParties.API/CustomMiddlewares/DatabaseRolesMiddleware.cs

```diff
@@ -20,10 +20,19 @@ public async Task InvokeAsync(HttpContext context, IMediator mediator)
         {
             var user = context.User;
 
+            var getCurrentUserQuery = new GetCurrentUserQuery(user);
```

Comment:
You need to rename middleware, because it is not only "Roles" middleware more. It is kind of `DatabaseAuthorizationMiddleware`.

Reply:
Class has been renamed